lib/srfi/26/expand-cut.scm

```diff
@@ -0,0 +1,46 @@
+(define (expand-cut id close? list<argument-or-slot>)
+
+  (define (gs s) (gensym (string->uninterned-symbol s)))
+
+  (let cons-closure ((l<argument-or-slot> list<argument-or-slot>)
```

Comment:
The use of angle brackets within the identifiers is distracting.  Moreover the use of `l` as an identifier (with or without the angle brackets) should be avoided because it is easy to confuse with a `1`.  `lst` would be preferable.